server/views/formPages/approvedView.html

```diff
@@ -0,0 +1,100 @@
+{% extends "./formTemplate.html" %}
+{% from "summary-list/macro.njk" import govukSummaryList %}
+
+{% block content %}
+
+<div class="govuk-grid-row govuk-body-s">
+  <div class="govuk-grid-column-two-thirds">
+
+    <h1 class="govuk-heading-l">Categorisation outcome</h1>
+
+    {% set cat = data.categoriser.provisionalCategory.suggestedCategory %}
+    {% set overriddenCat = data.categoriser.provisionalCategory.overriddenCategory %}
+    {% set categoriserCat = overriddenCat or cat %}
+    {% set superCat = data.supervisor.review.supervisorOverriddenCategory if data.supervisor.review.supervisorCategoryAppropriate == 'No' %}
```

Comment:
I thought we were having a common content section then separate top level pages for each role?